Formula/mytop.rb

```diff
@@ -1,101 +0,0 @@
-class Mytop < Formula
```

Comment:
We should deprecate this ASAP first before deletion.

Reply:
but there is no usage of this?

Reply:
I will do disable instead then (so that we dont need to worry about openssl migration)